bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
+
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+
+const proxy = httpProxy.createProxyServer({ secure: false });
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+console.log(`apiHost: ${apiHost}`);
+console.log(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
```

Comment:
Can we remove the user agent check? If we need to develop on the old desktop site, I think it's understandable to not use the proxy for that. You could just check for more URL prefixes like `developers`, `editors`, etc to support what we need from desktop during local development, right?